package com.cloudappstudio.utility;

import java.util.List;
import org.junit.*;
import static org.junit.Assert.*;
import com.cloudappstudio.data.CloudAuthId;
import com.cloudappstudio.data.CloudWebApp;
import android.os.Parcel;

/**
 * The class <code>CloudWebAppParserTest</code> contains tests for the class <code>{@link CloudWebAppParser}</code>.
 *
 * @author denmif
 * @version $Revision: 1.0 $
 */
public class CloudWebAppParserTest {
	/**
	 * Run the CloudWebAppParser(CloudAuthId) constructor test.
	 *
	 * @throws Exception
	 *
	 */
	@Test
	public void testCloudWebAppParser()
		throws Exception {
		CloudAuthId id = new CloudAuthId(Parcel.obtain());

		CloudWebAppParser result = new CloudWebAppParser(id);

		
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.RuntimeException: Stub!
		//       at android.os.Parcel.obtain(Parcel.java:5)
		assertNotNull(result);
	}

	/**
	 * Run the List<CloudWebApp> parseFromCloud() method test.
	 *
	 * @throws Exception
	 *
	 */
	@Test
	public void testParseFromCloud_1()
		throws Exception {
		CloudWebAppParser fixture = new CloudWebAppParser(new CloudAuthId(Parcel.obtain()));

		List<CloudWebApp> result = fixture.parseFromCloud();

		
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.RuntimeException: Stub!
		//       at android.os.Parcel.obtain(Parcel.java:5)
		assertNotNull(result);
	}

	/**
	 * Run the List<CloudWebApp> parseFromCloud() method test.
	 *
	 * @throws Exception
	 *
	 */
	@Test
	public void testParseFromCloud_2()
		throws Exception {
		CloudWebAppParser fixture = new CloudWebAppParser(new CloudAuthId(Parcel.obtain()));

		List<CloudWebApp> result = fixture.parseFromCloud();

		
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.RuntimeException: Stub!
		//       at android.os.Parcel.obtain(Parcel.java:5)
		assertNotNull(result);
	}

	/**
	 * Run the List<CloudWebApp> parseFromCloud() method test.
	 *
	 * @throws Exception
	 *
	 */
	@Test
	public void testParseFromCloud_3()
		throws Exception {
		CloudWebAppParser fixture = new CloudWebAppParser(new CloudAuthId(Parcel.obtain()));

		List<CloudWebApp> result = fixture.parseFromCloud();

		
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.RuntimeException: Stub!
		//       at android.os.Parcel.obtain(Parcel.java:5)
		assertNotNull(result);
	}

	/**
	 * Run the List<CloudWebApp> parseFromCloud() method test.
	 *
	 * @throws Exception
	 *
	 */
	@Test
	public void testParseFromCloud_4()
		throws Exception {
		CloudWebAppParser fixture = new CloudWebAppParser(new CloudAuthId(Parcel.obtain()));

		List<CloudWebApp> result = fixture.parseFromCloud();

		
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.RuntimeException: Stub!
		//       at android.os.Parcel.obtain(Parcel.java:5)
		assertNotNull(result);
	}

	/**
	 * Run the List<CloudWebApp> parseFromString(String) method test.
	 *
	 * @throws Exception
	 *
	 */
	@Test
	public void testParseFromString_1()
		throws Exception {
		CloudWebAppParser fixture = new CloudWebAppParser(new CloudAuthId(Parcel.obtain()));
		String json = "";

		List<CloudWebApp> result = fixture.parseFromString(json);

		
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.RuntimeException: Stub!
		//       at android.os.Parcel.obtain(Parcel.java:5)
		assertNotNull(result);
	}

	/**
	 * Run the List<CloudWebApp> parseFromString(String) method test.
	 *
	 * @throws Exception
	 *
	 */
	@Test
	public void testParseFromString_2()
		throws Exception {
		CloudWebAppParser fixture = new CloudWebAppParser(new CloudAuthId(Parcel.obtain()));
		String json = "";

		List<CloudWebApp> result = fixture.parseFromString(json);

		
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.RuntimeException: Stub!
		//       at android.os.Parcel.obtain(Parcel.java:5)
		assertNotNull(result);
	}

	/**
	 * Run the List<CloudWebApp> parseFromString(String) method test.
	 *
	 * @throws Exception
	 *
	 */
	@Test
	public void testParseFromString_3()
		throws Exception {
		CloudWebAppParser fixture = new CloudWebAppParser(new CloudAuthId(Parcel.obtain()));
		String json = "";

		List<CloudWebApp> result = fixture.parseFromString(json);

		
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.RuntimeException: Stub!
		//       at android.os.Parcel.obtain(Parcel.java:5)
		assertNotNull(result);
	}

	/**
	 * Run the List<CloudWebApp> parseFromString(String) method test.
	 *
	 * @throws Exception
	 *
	 */
	@Test
	public void testParseFromString_4()
		throws Exception {
		CloudWebAppParser fixture = new CloudWebAppParser(new CloudAuthId(Parcel.obtain()));
		String json = "";

		List<CloudWebApp> result = fixture.parseFromString(json);

		
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.RuntimeException: Stub!
		//       at android.os.Parcel.obtain(Parcel.java:5)
		assertNotNull(result);
	}

	/**
	 * Run the List<CloudWebApp> parseFromString(String) method test.
	 *
	 * @throws Exception
	 *
	 */
	@Test
	public void testParseFromString_5()
		throws Exception {
		CloudWebAppParser fixture = new CloudWebAppParser(new CloudAuthId(Parcel.obtain()));
		String json = "";

		List<CloudWebApp> result = fixture.parseFromString(json);

		
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.RuntimeException: Stub!
		//       at android.os.Parcel.obtain(Parcel.java:5)
		assertNotNull(result);
	}

	/**
	 * Run the List<CloudWebApp> parseFromString(String) method test.
	 *
	 * @throws Exception
	 *
	 */
	@Test
	public void testParseFromString_6()
		throws Exception {
		CloudWebAppParser fixture = new CloudWebAppParser(new CloudAuthId(Parcel.obtain()));
		String json = "";

		List<CloudWebApp> result = fixture.parseFromString(json);

		
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.RuntimeException: Stub!
		//       at android.os.Parcel.obtain(Parcel.java:5)
		assertNotNull(result);
	}

	/**
	 * Run the List<CloudWebApp> parseFromString(String) method test.
	 *
	 * @throws Exception
	 *
	 */
	@Test
	public void testParseFromString_7()
		throws Exception {
		CloudWebAppParser fixture = new CloudWebAppParser(new CloudAuthId(Parcel.obtain()));
		String json = "";

		List<CloudWebApp> result = fixture.parseFromString(json);

		
		// An unexpected exception was thrown in user code while executing this test:
		//    java.lang.RuntimeException: Stub!
		//       at android.os.Parcel.obtain(Parcel.java:5)
		assertNotNull(result);
	}

	/**
	 * Perform pre-test initialization.
	 *
	 * @throws Exception
	 *         if the initialization fails for some reason
	 *
	 */
	@Before
	public void setUp()
		throws Exception {
	}

	/**
	 * Perform post-test clean-up.
	 *
	 * @throws Exception
	 *         if the clean-up fails for some reason
	 *
	 */
	@After
	public void tearDown()
		throws Exception {
	}

	/**
	 * Launch the test.
	 *
	 * @param args the command line arguments
	 *
	 */
	public static void main(String[] args) {
		new org.junit.runner.JUnitCore().run(CloudWebAppParserTest.class);
	}
}